api/nakadi-event-bus-api.yaml

```diff
@@ -400,7 +746,7 @@ definitions:
   Event:
     type: object
     description: |
-      This is the most general representation of an event, that can be processed
+      **To be changed**. This is the most general representation of an event, that can be processed
```

Comment:
looking on the line 757, the example of the `event_type` value should changed according to our `:` scoped suggested format